tests/test_promotion.py

```diff
@@ -212,6 +216,19 @@ def test_find_by_discount_ratio(self):
         self.assertEqual(promotions[0].product_id, 9527)
         self.assertEqual(promotions[0].name, "20%OFF")
 
+    def test_redeem_promotion(self):
+        """ Redeem a Promoion """
+        Promotion(name="20%OFF", product_id=9527, discount_ratio=0.8).save()
+        promotion = Promotion.find(1)
+        self.assertEqual(promotion.counter, 0)
+        Promotion.redeem_promotion(1)
+        promotion = Promotion.find(1)
```

Comment:
I think line 225 is redundant.

Reply:
Yes. This may have something to do with the internal implementation of the database. I will make the change.

Ref:
- [the sqlalchemy-api](http://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.get)